src/common/desktop/export/DesktopExportFacade.ts

```diff
@@ -84,6 +86,11 @@ export class DesktopExportFacade implements ExportFacade {
 	}
 
 	async startMailboxExport(userId: string, mailboxId: string, mailBagId: string, mailId: string): Promise<void> {
+		if (this.lockedUserIds.has(userId)) {
+			throw new LockedError(`Export is locked for user: ${userId}`)
```

Comment:
`LockedError` is one of the API errors, I'm not sure if it's a good idea to reuse it. It's probably fine but I'm afraid we can mix it up with some instance being locked.